### PR TITLE
add coverId param to getPayoutOutcome

### PR DESCRIPTION
**Pull request by danoctavian**

### CODE REVIEW

File: test/integration/ClaimPayoutAddress/payouts.js [hunk 1]
```diff
@@ -85,50 +85,7 @@ describe('send claim payout to the payout address', function () {
 
     assert(actualPayout.eq(expectedPayout), 'should have transfered the cover amount');
   });
-
-  it('[A1, status: 0, 7, 14] CA accept, closed with closeClaim()', async function () {
```
Comment (danoctavian) [inline test/integration/ClaimPayoutAddress/payouts.js:89]: mistake from prev branch added redundant test for debugging, removing it here

File: contracts/modules/cover/Cover.sol [hunk 2]
```diff
@@ -53,8 +53,8 @@ contract Cover is MasterAware {
   );
```
Comment (shark0der) [inline contracts/modules/cover/Cover.sol:53]: Index `uint coverId` as well

Comment (shark0der) [reply contracts/modules/cover/Cover.sol:53]: Should we index `coverAsset` as well? So we can filter quickly like "fetch all DAI covers"...?

Comment (danoctavian) [reply contracts/modules/cover/Cover.sol:53]: the limitation here is only 3 indexed fields i already have 3 

    address indexed buyer,
    address indexed contractAddress,
        CoverType indexed coverType,
        
        
        can swawp coverType out for coverAsset would make sense, but can't do more than that

**Pull request merged**